docs/getting-started/03-deploy-microservice.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Deploy a microservice
+type: Getting Started
+---
+
+You will now start by deploying a standalone [`orders-service`](https://github.com/kyma-project/examples/blob/master/http-db-service/README.md) microservice in the `orders-service` Namespace. This microservice will act as a link between the external application and the Redis service and we will build the whole end-to-end flow around it.
```

Comment:
```suggestion
You will deploy a standalone [`orders-service`](https://github.com/kyma-project/examples/blob/master/http-db-service/README.md) microservice in the `orders-service` Namespace. This microservice will act as a link between the external application and the Redis service and we will build the whole end-to-end flow around it.
```
**Reason:**
`You will now start by (...)` suggest that this is just the first step to take, while in reality this is the goal that we want to achieve with this guide. 
I believe this sentence means to introduce the reader into the purpose of this guide and therefore simply telling them what they'll do in this guide using `You will now deploy (...)` seems more suitable here.

Reply:
I mean it in the context of the whole set - in this sense this guide is only a start, but again (same as before), if you think that might be unclear, sure I will change that.